postinst

```diff
@@ -0,0 +1,21 @@
+#!/bin/sh
+
+# Enable on boot
+/etc/init.d/wrtbwmon enable
+
+# Configure our cron
+if [ -d "/etc/crontabs" ]; then
+  echo "PATH=/usr/sbin:/usr/bin:/sbin:/bin" > wrtbwmon
```

Comment:
So the concept is that wrtbwmon will be in /etc/crontabs to run as a cron service for wrtbwmon. Looks like I forgot to define the dir, so will fix this as well.